entrypoint/common.js

```diff
@@ -0,0 +1,139 @@
+// Copyright 2020 Carton contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { SwiftRuntime } from "javascript-kit-swift";
+import { WASI } from "@wasmer/wasi";
+import { WasmFs } from "@wasmer/wasmfs";
```

Comment:
We don't need to use Wasmer's polyfill on Node.js because it also has builtin WASI implementation.

Reply:
I am afraid that the last time I tried running with the native Node WASI, the tests crashed. I think it could maybe be something interesting in the future, though.  My concerns doing that are:

- Node's WASI is still in an experimental stage
- Tests will use a different WASI implementation than the one used in production. Using the Wasmer's polyfill in Node has allowed us in the past to detect errors like the `clock_rest_get` memory refresh (btw kudos to you, @kateinoigakukun, for the help there)

What do you think?

Reply:
> - Node's WASI is still in an experimental stage

My impression is Node's WASI is more stable, maintained, and well-tested than wasmer's one.
For example, wasmer's one doesn't support reactor model, and it's recently replaced with an entirely new implementation since version 1.x but it's still not production-ready quality. So I prefer Node.js's one personally.

> Tests will use a different WASI implementation than the one used in production.

OK, it seems reasonable to keep wasmer's WASI implementation while using it in the browser environment.

Let's investigate the crash and add options to switch WASI backends in another PR.

Reply:
@kateinoigakukun do you have anything that you'd like to see addressed in this specific PR? If not, can it get your approval then?